src/pudl/helpers.py

```diff
@@ -421,7 +422,7 @@ def cleanup(df, on, by):
 
 def get_pudl_dtype(col, data_source):
     """Look up a column's canonical data type based on its PUDL data source."""
-    return pc.COLUMN_DTYPES[data_source][col]
+    return get_pandas_dtypes(group=data_source)[col]
```

Comment:
Type hints would be nice here.

Reply:
I think it would probably be better to just replace all uses of this function, since it's just a one-line wrapper. But I was trying to change as little as possible while still being able to rip out `pc.COLUMN_DTYPES`

Reply:
Okay, I chased down a bunch of these calls and also `convert_cols_dtypes` and replaced them with `pudl.metadata.fields.apply_pudl_dtypes`. Much less column contents munging going on now everywhere. See commit messages.

Reply:
`apply_pudl_dtypes()` looks great. Excited to use it.